Extensions/xkit_patches.js

```diff
@@ -201,6 +232,13 @@ XKit.tools.make_gist = function(text, name) {
 	});
 };
 
+/**
+ * Injects a function into the DOM scope.
```

Comment:
I'm not sure if "DOM scope" is a very precise name for what we're doing here, maybe "page context" would be better? can't remember offhand how its referred to.

alternative wording:

> Copies a function from the addon context into the page context. This function will be serialized to a string, and then injected as a script tag into the page. `addt` will be serialized as JSON and then passed as a parameter to the function